src/components/addTask.js

```diff
@@ -96,78 +111,71 @@ function AddTask({ user }) {
+                <div class="control " id="eventnamecontrol">
                   <input
-                    className="input is-secondary "
+                    class="input is-secondary "
                     type="text"
                     placeholder="event name"
                     id="eventname"
                     style={{ color: "black" }}
+                    oninvalid = "this.setCustomValidity('Please enter an event name')"
+                    oninput = "setCustomValidity('')"
+                    required
                   />
                 </div>
-                <p className="helpName"> </p>
+                <p class="helpName"> </p>
               </div>
-            </form>
-            <form style={{ marginTop: 20 }}>
-              <div className="field">
-                <label className="label "> Event Description</label>
+              <div class="field">
+                <label class="label "> Event Description</label>
                 <div
-                  className="control has-icons-left"
+                  class="control has-icons-left"
                   id="eventdescriptioncontrol"
                 >
                   <textarea
-                    className="textarea"
+                    class="textarea"
                     type="text"
                     placeholder="Event Description"
                     id="eventdescription"
                     style={{ color: "black" }}
+                    required
                   />
                 </div>
               </div>
-            </form>
-            <form style={{ color: "black" }}>
-              <div className="field" style={{ marginTop: 20 }}>
-                <label className="label"> Event Date</label>
-                <form>
-                  <section
-                    className="field is-rounded"
-                    style={{ backgroundColor: "white", width: "238px" }}
-                  >
-                    <DateTimePicker
-                      id="datetimepicker"
-                      disableClock={true}
-                      disableCalendar={true}
-                      yearPlaceholder="yyyy"
-                      monthPlaceholder="mm"
-                      dayPlaceholder="dd"
-                      hourPlaceholder="00"
-                      minutePlaceholder="00"
-                      maxDate={maxDate}
-                      onChange={(date) => setSelectedDate(date)}
-                      isClearable
-                    />
-                  </section>
-                </form>
+              <div class="field" style={{ marginTop: 20 }}>
+                <label class="label"> Event Date</label>
+                <section
+                  class="field is-rounded"
+                  style={{ backgroundColor: "white", width: "205px" }}
+                  has-backgroundColor
+                >
+                  <DateTimePicker
+                    id="datetimepicker"
+                    disableClock="true"

```

Comment:
`disableClock` takes a boolean, but you're passing it a string here. Needs to be `disableClock={true}`. (Same for `disableCalendar`)